operator/jsonnet/obs-operator.jsonnet

```diff
@@ -8,7 +8,7 @@ local patchObs = obs {
   compact+::
     t.compact.withVolumeClaimTemplate {
       config+:: obs.compact.config,
-    },
+    } + (if std.objectHas(obs.compact.config, 'enableDownSampling') && obs.compact.config.enableDownSampling == false then t.compact.withDownsamplingDisabled else {}),
```

Comment:
This configuration defaults to having downsampling enabled, but this is not what the CRD field name implies. Let's flip the logic here so that you need to set the EnableDownsampling field to true for it to actually be enabled.